zookeeper-server/src/main/java/org/apache/zookeeper/server/persistence/FileSnap.java

```diff
@@ -78,15 +78,17 @@ public long deserialize(DataTree dt, Map<Long, Integer> sessions) throws IOExcep
             return -1L;
         }
         File snap = null;
+        long snapZxid = 0;
```

Comment:
should the default value of `snapZxid` be -1 instead of 0? -1 is what `Util.getZxidFromName` returns if finds nothing valid.

Reply:
There is no correctness issue with 0, but I'll change it to -1 to make it identical with the default returned from Util.getZxidFromName.